megamek/src/megamek/client/ui/swing/boardview/BoardView1.java

```diff
@@ -2342,25 +2342,28 @@ private void drawHexes(Graphics g, Rectangle view) {
      */
     private void drawHexes(Graphics g, Rectangle view, boolean saveBoardImage) {
         // only update visible hexes
-        int drawX = (int) (view.x / (HEX_WC * scale)) - 1;
-        int drawY = (int) (view.y / (HEX_H * scale)) - 1;
-
-        int drawWidth = (int) (view.width / (HEX_WC * scale)) + 3;
-        int drawHeight = (int) (view.height / (HEX_H * scale)) + 3;
+        double xs = (int) (HEX_WC*scale);
```

Comment:
if `xs` and `ys` are declared as a `double`, why are you casting them to an `int`?